src/Nether.Analytics/IMessageRouter.cs

```diff
@@ -7,7 +7,7 @@ namespace Nether.Analytics
 {
     public interface IMessageRouter
     {
-        //TODO: Make method Async
-        Task RouteMessageAsync(Message msg);
+        Task RouteMessageAsync(string partitionId, Message msg);
```

Comment:
Why is there the new partitionId parameter? The Message type that is passed in has a PartitionId property